WORKSPACE.bazel

```diff
@@ -266,3 +266,27 @@ sns_quill(name = "sns_quill")
 load("//bazel:idl2json.bzl", "idl_to_json")
 
 idl_to_json(name = "idl2json")
+
+# Use libsystemd from the host environment
+# Ideally this is pulled hermetically, but as this is only used by infogetty
+# within IC images, we have less compatibility to worry about, and packaging it
+# this way is easier.
+new_local_repository(
+    name = "libsystemd",
+    build_file_content = """
+cc_import(
+    name = "libsystemd-internal",
+    hdrs = glob(["include/systemd/*.h"]),
+    interface_library = "lib/x86_64-linux-gnu/libsystemd.so",
```

Comment:
do we need the full lib or can we get away just with the header(s)?

Reply:
From what I remember, the headers will take care of compiling, but the `.so` is still needed at build time to set up the dynamic link.

Reply:
ah sorry, didn't realize we were calling a function and thought we were just using definitions like `LOG_ERR`

Reply:
actually, you might be able to get away with just the headers, looks like the function is just `printf` with some `#define`s: https://github.com/systemd/systemd/blob/338553715b618aba8299d053a8796f7ce9c58d3e/src/systemd/_sd-common.h#L30

Reply:
It looks possible to lift the function, but involved. Is this what you had in mind? https://github.com/systemd/systemd/blob/1f5d8a6132f12574f524cef72dbda0f7408c4217/src/libsystemd/sd-journal/journal-send.c#L102-L152